docs/getting-started/usage.md

```diff
@@ -54,7 +54,7 @@ As you can see, Chart.js requires minimal markup: a `canvas` tag with an `id` by
 Lastly, let’s create the `src/acquisitions.js` file with the following contents:
 
 ```jsx
-import { Chart } from 'chart.js/auto'
```

Comment:
Named export `Chart` also exists there, so it is also a working code.

Reply:
Yes but in the installation section we use this syntax since v3 so to be more consistent i like this approach more